test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8872,6 +8872,39 @@ def forward(self, x):
         self.run_test(M(2, 1), (x,))
         self.run_test(M([-1, 3], [-2, -1]), (x,))
 
+    @skipIfUnsupportedMinOpsetVersion(13)
+    def test_sequence_to_float(self):
+        class M(torch.nn.Module):
+            def forward(self, x):
+                result = torch.tensor([1.1 for _ in range(x.shape[1])], dtype=torch.float)
```

Comment:
If I understand correctly, this converts to graph like
```
%seq = onnx::Loop(...)
%vec = onnx::ConcatFromSequence(%seq, ...)
```

Is `onnx::ConcatFromSequence` further optimized away, that `%seq` optimized as scan output of loop?

Reply:
An ONNX graph example is:

graph(%x.1 : Long(10, 5, strides=[5, 1], requires_grad=0, device=cpu)):
  %1 : Double(requires_grad=0, device=cpu) = onnx::Constant[value={1.1}]()
  %3 : Bool(requires_grad=0, device=cpu) = onnx::Constant[value={1}]()
  %7 : float[] = prim::ListConstruct()
  %8 : Long(2, strides=[1], device=cpu) = onnx::Shape(%x.1)
  %10 : Long(device=cpu) = onnx::Constant[value={1}]()
  %11 : Long(device=cpu) = onnx::Gather[axis=0](%8, %10) 
  %12 : float[] = onnx::Loop(%11, %3, %7) 
    block0(%13 : Long(device=cpu), %cond : bool, %14 : float[]):
      %15 : float[] = onnx::SequenceInsert(%14, %1) 
      %17 : Bool(requires_grad=0, device=cpu) = onnx::Identity(%3)
      -> (%17, %15)
  %18 : Tensor = onnx::ConcatFromSequence[axis=0, new_axis=1](%12)
  %19 : FloatTensor = onnx::Cast[to=1](%18) 
  %20 : FloatTensor(device=cpu) = onnx::Add(%x.1, %19) 
  return (%20)

I don't see the optimization happened. Is it supposed to happen? If so, probably I could go deep to check why it didn't happen in this case to see if it is a bug.